src/test/resources/net/sf/jabref/exporter/MsBibExportFormatTest1.bib

```diff
@@ -0,0 +1,44 @@
+@unpublished{,
+  author =                      {Hanns-Jörg Stoß},
+  month =                       {July},
+  note =                        {a Note},
+  abstract =                    {Es werdenTuring-Maschinen mitk unabhängigen Lese-Schreib-Köpfen auf einem Band undTuring-Maschinen mit je einem Kopf aufk Bändern betrachtet. Es wird gezeigt, daß sich ohne Verändern der Rechenzeit jede Maschine von einem Typ durch eine Maschine vom andern Typ simulieren läßt. Daraus wird gefolgert, daß es für die Rechengeschwindigkeit vonTuring-Maschinen nur auf die Zahl der Köpfe, nicht aber auf deren Verteilung auf einzelne Bänder ankommt.},
```

Comment:
The problem with using the database is, that when there are more than one entry, the order of the elements in the exported xml file is somehow changing everytime. So I cannot make one xml document for all entries in the database, but have to make a xml file for every entry. So if you're fine with it, I wouldn't use the database here.